ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -101,8 +95,20 @@ class VariableProcessorTest {
     }
 
     @Test
-    fun `process variables processes sub_period`() {
-        expectVariablesResult("{{ sub_period }}", "PERIOD_NAME")
+    fun `process variables processes sub_period annual`() {
+        expectVariablesResult("{{ sub_period }}", "Annual")
+    }
+
+    @Test
+    fun `process variables processes sub_period monthly`() {
+        rcPackage = TestData.Packages.monthly
+        expectVariablesResult("{{ sub_period }}", "Monthly")
+    }
+
+    @Test
+    fun `process variables processes custom period`() {
+        rcPackage = TestData.Packages.annual.copy(packageType = PackageType.CUSTOM)
+        expectVariablesResult("{{ sub_period }}", "")
```

Comment:
Just want to confirm this is what we expect @NachoSoto

Reply:
Yeah 👍🏻  I don't have a test for this in iOS but this is fine.
The idea is to have validations in the frontend / backend to prevent users from using these with custom pacakges.